sherpa/astro/plot.py

```diff
@@ -473,16 +473,21 @@ class RMFPlot(shplot.HistogramPlot):
         The labels for each line.
     """
 
-    histo_prefs = shplot.basicbackend.get_model_histo_defaults()
+
+    rmf_plot_prefs = shplot.basicbackend.get_rmf_plot_defaults()
 
     # TODO: Make that a plot preference
     # How many monochromatic lines to use
     n_lines = 5
 
     # Because this derived from NoNewAttributesAfterInit we need to
-    # make the attribute here it cna be used in prepare
+    # make the attribute here it can be used in prepare
     labels = None
 
+
+    def _merge_settings(self, kwargs):
+        return {**self.rmf_plot_prefs, **kwargs}
```

Comment:
I guess I can get back at you and say this can be (I think)

```suggestion
        return self.rmf_plot_prefs | kwargs
```

although is there actually a subtle difference between the two since the original version creates a new dictionary whereas `a | b` will return a if b is empty (and so changes to the return value would then get passed back to `self.rmf_plot_prefs` and could that be an issue. hmm.

Reply:
When we redo the way preferences work, we'll address that.